### ISSUE TITLE
API less way to allow software to check for new releases.

### ISSUE THREAD

**Issue by AraHaan** [isaacs/github#1417]
what I would like is for user software to be able to get the latest release version (the one released the most recently in case the name of the release does not list it first) like to a place like for example:

``https://github.com/<user>/<repo>/releases/latest``

Or a way to get that from software without the github api, so the software would know that version: 0.16.0 or whatever was just released and the software itself can download it and auto apply the said update (like what most professional software does at least on Windows like Windows downloding it’s own updates).

**Comment by mattyoho** [comment]
Howdy. :wave: I think the feature you're asking for may already exist. 🙂

For example, visiting `https://github.com/rbenv/rbenv/releases/latest` will (currently) redirect to `https://github.com/rbenv/rbenv/releases/tag/v1.1.1`.

Is this what you had in mind? Thanks!

**Comment by AraHaan** [comment]
yes thanks, but where it is the most recently (time wise) created.
